Configure

```diff
@@ -1387,7 +1387,7 @@ else               { $no_user_defines=1;    }
 
 unless ($disabled{afalgeng}) {
     $config{afalgeng}="";
-    if ($target =~ m/^linux/) {
+    if ($target =~ m/^linux/ || $target{afalgeng} eq "enable") {
```

Comment:
Check `grep { $_ eq 'afalgeng' } @{$target{enable}}` instead, and have this in the config target:

``` perl
        enable      => [ 'afalgeng' ],
```

(actually, it would be great if you added that in `linux-generic32` as well)

Reply:
With that, the check `$target =~ m/^linux/` can be removed entirely